yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Nodes and Actors
+---
+
+To analyze the suitability of different node types for various actor types in the system, let's break down the node types and actor types, considering the characteristics, preferences, abilities, and potential motivations of each actor. This will help determine which actors are likely to run which nodes and how they might interact with the Private eXecution Environment (PXE).
+
+## Node Types
+
+The below node types are ordered in resource requirements. The types exists both in the context of Aztec but also other blockchains. All the nodes, participate in the peer-to-peer (p2p) network, but with varying capacity.
+
+1. **Light Node**: 
+    - Follows the blockchain but only interested in headers (stores headers).
```

Comment:
Since we have 3 blockchains (the Aztec L2 p2p network, the Ethereum p2p network, and the DA p2p network), we should specify which of those p2p networks each of these participants needs to track. Do they need to run a node for all 3 networks? If not, which do they need to run, etc.

Reply:
Edit, I see a section below on Ethereum nodes. Which kinds of ethereum nodes does a user need to run, if they want to run a particular type of aztec node? Or maybe it's easier to say what type of ethereum node is _not_ sufficient, if you want to run a particular type of aztec node? This info could perhaps form part of the table.